packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -3288,6 +3346,49 @@ void main() {
     expect(tester.testTextInput.setClientArgs!['obscureText'], isFalse);
   });
 
+  testWidgets('Sends viewId and updates config when it changes', (WidgetTester tester) async {
+    int viewId = 14;
+    late StateSetter setState;
+    final GlobalKey key = GlobalKey();
+
+    await tester.pumpWidget(
+      wrapWithView: false,
+      StatefulBuilder(
+        builder: (BuildContext context, StateSetter stateSetter) {
+          setState = stateSetter;
+          return View(
+            view: FakeFlutterView(tester.view, viewId: viewId),
+            child: MediaQuery(
+              data: const MediaQueryData(),
+              child: Directionality(
+                textDirection: TextDirection.ltr,
+                child: EditableText(
```

Comment:
I think this test doesn't quite cover the case that I mentioned above about viewId changing, because the EditableText will be rebuilt on setState.

Reply:
I'm giving a `GlobalKey` to `EditableText` for this specific reason. I also confirmed (using a print) that we are actually calling `updateConfig`. I think it makes sense to add a check in the test to make sure that an `updateConfig` was actually sent.